packages/generator/src/generators/app-generator.ts

```diff
@@ -267,17 +267,7 @@ export class AppGenerator extends Generator<AppGeneratorOptions> {
       ["git", ["add", "."], {stdio: "ignore"}],
       [
         "git",
-        [
-          "-c",
-          "user.name='Blitz.js CLI'",
-          "-c",
-          "user.email='noop@blitzjs.com'",
-          "commit",
-          "--no-gpg-sign",
-          "--no-verify",
-          "-m",
-          "Brand new Blitz app!",
-        ],
+        ["commit", "--no-gpg-sign", "--no-verify", "-m", "Brand new Blitz app!"],
```

Comment:
Since we're changing it to use the user's git credentials, I'm not sure we still want `--no-gpg-sign`. Are you sure it's still preferred?

Reply:
I believe you're right. In general we should be using the user's local config regardless.